src/routers/comment-router.js

```diff
@@ -0,0 +1,95 @@
+const express = require("express");
+const router = express.Router();
+const { check, validationResult } = require("express-validator");
+const Comment = require("../models/Comment");
+const comment_controller = require("../controllers/comment-controller");
+
+//Comment Routes
+
+/*
+Routes GET requests at root 
+returns a promise of a JSON object containing a list of comments
+*/
+router.get("/", function(req, res) {
+  const { userID, username, text, date } = req.body;
+
+  Comment.find({
+    $or: [{ userID: userID }, { text: text }, { date: date }]
+  })
+    .lean()
+    .exec(function(err, comments) {
+      if (err) {
+        {
+          res.status(500).send(err);
+        }
+      } else {
+        res.send(JSON.stringify(comments));
+      }
+    });
+});
+
+/*
+Routes Post requests at "/create" 
+Creates new comment document from request body
+Populates username of comment document
+Sends JSON of comment back in response 
+*/
+
+router.post("/create", async function(req, res) {
```

Comment:
You can leave the it as just `"/"`, no need for `/create`, only the `.post` can create